day01/index.js

```diff
@@ -0,0 +1,43 @@
+const R = require('ramda');
+const fs = require('fs');
+const path = require('path');
+
+const inputData = fs.readFileSync(path.resolve(__dirname, 'data.txt'), 'utf8');
+
+const convertInputToArr = input =>
+  input
+    .toString()
+    .split(/\n|\$/)
+    .map(item => parseInt(item, 10));
```

Comment:
I know this example is small but I really think this is a job for another module and not responsibility of the fuel mass calculation. I would move it out. The benefit, as always, is that your core alogrithm doesn't rely on I/O type things. Getting the data out of a mushy string and into a clearly defined data structure (like your array, here) will always pay you back large dividends when it comes time to get data from other places.

Reply:
Hey @gtempus , I have addressed this in ceacc2